src/stdlib/base.bt

```diff
@@ -145,17 +145,20 @@ macro comm($pid) {
     }
 
     let $name : string[64];
-    let $ret = (int32)__bpf_task_comm_from_pid((int32)$pid, &$name);
-    // EOPNOTSUPP: Operation not supported
-    if ($ret == (int32)-95) {
-      warnf("'bpf_task_from_pid kfunc', which is used for getting the process comm, is not supported on your system.");
-      $name = "[unknown]";
-    // ESRCH: No such process
-    } else if ($ret == (int32)-3) {
-      warnf("No such process (pid=%d)", $pid);
-      $name = "N/A";
+    if comptime __builtin_kfunc_allowed("bpf_task_from_pid") {
```

Comment:
Can we make friendlier user-facing macro wrappers for this e.g. `kfunc_allowed` and `kfunc_exists` ?